srv/exo/bin/_startDataBase.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -eu
+
+# #############################################################################
+# Initialize
+# #############################################################################                                              
+SCRIPT_NAME="${0##*/}"
+SCRIPT_DIR="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Load env settings
+source ${SCRIPT_DIR}/_setenv.sh
+# Load common functions
+source ${SCRIPT_DIR}/_functions.sh
+
+echo "[INFO] ======================================="
+echo "[INFO] Starting Database server on ${HOSTNAME}..."
+echo "[INFO] ======================================="
+
+if [ -e /lib/systemd/system/mysql.service ]; then
```

Comment:
Is there a particular reason to test the presence of this file ?
the script should failed if the systemctl command failed